queue_services/entity-auth/devops/vaults.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "vault": "database",
+        "application": [
+            "entity-db"
+        ]
+    },
+    {
+        "vault": "sentry",
+        "application": [
+            "entity"
+        ]
+    },
+    {
+        "vault": "entity",
+        "application": [
+            "entity-auth"
```

Comment:
we might need to create a new application. we can reuse if its available in a different vault

Reply:
yeah, a new application makes sense I think